src/linalg/schur.rs

```diff
@@ -413,7 +414,6 @@ where
             let inv_rot = rot.inverse();
             inv_rot.rotate(&mut m);
             rot.rotate_rows(&mut m);
-            m[(1, 0)] = N::zero();
```

Comment:
Is this intentional? This reverts your change introduced by #668.

Reply:
Sorry, my bad. The creation of a new pull request used the nalgebra/dev branch not mine.
I should have fixed the issue.